marimo/_plugins/ui/_core/ui_element.py

```diff
@@ -470,9 +470,18 @@ def _on_update_completion(self) -> bool:
     def __deepcopy__(self, memo: dict[int, Any]) -> UIElement[S, T]:
         # Custom deepcopy that excludes elements that can't be deepcopied
         cls = self.__class__
+        return self._from_args(memo, self.__dict__, cls=cls, args=self._args)
+
+    def _from_args(
```

Comment:
should this be static or a cls method?

Reply:
Uses an instance, but not in the way I hook in. Changed, and realized I had some dead code